test/Table_Tests/src/Common_Table_Operations/Nothing_Spec.enso

```diff
@@ -0,0 +1,240 @@
+                table.at "x" . is_in (table.at "z" . to_vector) . to_vector . should_equal [true_if_not_nothing, Nothing]
+
+            Test.specify "Correctly handle Nothing in: "+value_type.to_text+".is_in Vector with Nothings (returning True)" <|
+                table.at "x" . is_in (table.at "x" . to_vector) . to_vector . should_equal [true_if_not_nothing, Nothing]
+
+            Test.specify "Correctly handle Nothing in: "+value_type.to_text+".is_in Vector with Nothings (returning Nothing)" <|
+                table.at "x" . is_in (table.at "y" . to_vector) . to_vector . should_equal [Nothing, Nothing]
+
+            Test.specify "Correctly handle Nothing in: Nothing.is_in Vector with Nothings (returning Nothing)" <|
+                table.at "n" . is_in (table.at "x" . to_vector) . to_vector . should_equal [Nothing, Nothing]
+
+            Test.specify "Correctly handle Nothing in: "+value_type.to_text+".is_in empty Vector (returning False)" <|
+                table.at "x" . is_in [] . to_vector . should_equal [False, False]
+
+    Test.group prefix+"Boolean is_in" <|
+        make_containing_values had_null had_true had_false =
+            null_maybe = if had_null then [Nothing] else []
+            true_maybe = if had_true then [True] else []
+            false_maybe = if had_false then [False] else []
+            null_maybe + true_maybe + false_maybe
+
+        bools = [True, False]
+        bools_and_nothing = [True, False, Nothing]
+        # Format: [input, had_null, had_true, had_false, output]
+        cases = bools_and_nothing.flat_map input-> bools.flat_map had_null-> bools.flat_map had_true-> bools.map had_false->
+            output = case input of
+                True -> if had_true then True else (if had_null then Nothing else False)
+                False -> if had_false then True else (if had_null then Nothing else False)
+                Nothing -> if had_null || had_true || had_false then Nothing else False
+            [input, had_null, had_true, had_false, output]
+
+        # To test negated columns, we invert the values before putting them into the Storage, and set the negated bit.
+        negate : Vector | Column -> Vector | Column
+        negate vorc = case vorc of
+            _ : Vector -> vorc.map (x-> x.if_not_nothing x.not) # Invert values first
+            _ : Column -> vorc.not                              # Set negated bit
+            _ : Database_Column -> vorc.not                     # Set negated bit
+        negation_cases = [["normal input, normal argument", identity, identity]]
+            + [["negated input, normal argument", negate, identity]]
+            + [["normal input, negated argument", identity, negate]]
+            + [["negated input, negated argument", negate, negate]]
+
+        negation_cases.map negation_case->
+            negation_desc = negation_case.at 0
+            neg_input = negation_case.at 1
+            neg_argument = negation_case.at 2
```

Comment:
nitpick but IMO this is misleading `neg_input` may be `negate` but it may as well be `identity`. So the `neg_` is misleading what this is.

Maybe just `transform_input` instead?

```suggestion
            transform_input = negation_case.at 1
            transform_argument = negation_case.at 2
```

Reply:
Done.